packages/grid_client/src/client.ts

```diff
@@ -185,15 +184,17 @@ class GridClient {
       await send("get", urlJoin(urls.rmbProxy, "version"), "", {});
     } catch (err) {
       console.log(err.message);
-      throw Error("failed to connect to Grid proxy server");
+      (err as Error).message = formatErrorMessage("Failed to connect to Grid proxy server", err);
+      throw err;
```

Comment:
Can't we throw a general error here?

Reply:
no, this handled in send function, if there is an error will be thrown as RequestError